pkg/cmd/roachtest/cluster.go

```diff
@@ -114,7 +114,7 @@ func validateBinaryFormat(path string, arch vm.CPUArch, checkEA bool) (string, e
 	}
 	if checkEA {
 		// Check that the binary was compiled with assertions _enabled_.
-		cmd = exec.Command("bash", "-c", fmt.Sprintf("%s version |grep \"Enabled Assertions\" |grep true", abspath))
+		cmd = exec.Command("bash", "-c", fmt.Sprintf("%s %s version |grep \"Enabled Assertions\" |grep true", install.SuppressMetamorphicConstantsEnvVar(), abspath))
```

Comment:
For my own understanding: the env var here shouldn't be necessary for correctness, right?